build/build.go

```diff
@@ -554,7 +556,22 @@ func (db *DeploymentBuilder) buildDataNetworks(d *Deployment) error {
 	return nil
 }
 
+func (db *DeploymentBuilder) GetAssociatedAddressPools(networkName string, networkAddressPool []networkAddressPools.NetworkAddressPool, addressPools []addresspools.AddressPool) []addresspools.AddressPool {
+	var associated_address_pools []addresspools.AddressPool
+	for _, network := range networkAddressPool {
```

Comment:
Following is misleading:
`for _, network := range networkAddressPool`

Can you change it to `for _, networkAddressPool := range networkAddressPools` instead?